WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -492,6 +488,31 @@ private void updateConnectionBar(boolean isConnected) {
         }
     }
 
+    private void handleBlogRemoved() {
+        if (!AccountHelper.isSignedIn()) {
+            ActivityLauncher.showSignInForResult(this);
+        } else {
+            MySiteFragment mySiteFragment = getMySiteFragment();
+            if (mySiteFragment != null) {
+                List<Map<String, Object>> allBlogs = WordPress.wpDB.getAllBlogs();
+                if (allBlogs.isEmpty()) {
```

Comment:
This is very nitpicky, so please feel free to ignore. I was just wondering if it would make sense to add a method in `WordPressDB` which will just return the number of blogs. This particular path will be used very rarely, if at all, since it's deleting a site, but if we need similar checks in other places, I thought it might give a small performance boost.